quarkchain/cluster/client.py

```diff
@@ -0,0 +1,60 @@
+from __future__ import print_function
+
+# import random
+import logging
+import sys
+import grpc
+
+import grpc_client_pb2_grpc
+import grpc_client_pb2
+
+
+class GrpcClient(object):
+    def __init__(self, channel):
+        self.channel = channel
+        self.stub = None
+
+    def set_root_chain_confirmed_block(self) -> bool:
+        if self.channel == None:
```

Comment:
prefer we don't allow constructor argument to be None on line 14

Reply:
Done.